build.sbt

```diff
@@ -24,6 +24,7 @@ lazy val alpakka = project
     sns,
     sqs,
     sse,
+    eip,	
```

Comment:
I think we should try to put the sample code under `doc/src/test/{java/scala}`  rather than introducing a new module just for samples

Reply:
I though about this. Then I started thinking how I would implement some other patterns (like the Enricher: http://www.enterpriseintegrationpatterns.com/patterns/messaging/DataEnricher.html) and realized it wont probably work out of the box. (I actually started working on this but it's not ready yet). So I started a new module as I needed to put the classes somewhere. Suggestions are welcome. (I can put these in docs and see afterwards when I finish the Enricher). Your call

Reply:
In Akka core that is how we do it when there is none of the actual modules that naturally would host the sample code, so I think we should follow that here.

Reply:
ok moved everything to docs. (Travis seems to fail randomly :( )